calibr8/utils.py

```diff
@@ -208,6 +214,66 @@ def plot_t_band(ax, independent, mu, scale, df, *, residual_type: typing.Optiona
     return artists
 
 
+def plot_continuous_band(ax, independent, model, residual_type: typing.Optional[str]=None):
+    """Helper function for plotting the 84, 95 and 97.5 % likelihood-bands of a univariate distribution.
+    
+    Parameters
+    ----------
+    ax : matplotlib.Axes
+        subplot object to plot into
+    independent : array-like
+        x-values for the plot
+    model : CalibrationModel
+        A fitted calibration model with data.
+        The predict_dependent method should return a tuple where the mean is the first entry.
+    residual_type : str, optional
+        One of { None, "absolute", "relative" }.
+        Specifies if bands are for no, absolute or relative residuals.
```

Comment:
```suggestion
        Specifies if bands should be plotted directly (`None`),
        with respect to the median (`"absolute"`),
        or `"relative"` to the absolute value of the median.
```

Reply:
It's not the median but rather the residual with respect to the median. I will change the suggestion accordingly.